modin/pandas/test/test_io.py

```diff
@@ -2398,15 +2399,15 @@ def test_read_fwf_s3(self, storage_options):
 
 
 class TestGbq:
-    @pytest.mark.xfail(reason="Need to verify GBQ access")
+    @pytest.mark.skip(reason="Can not pass without GBQ access")
```

Comment:
Why is this changed to `skip`?

Reply:
Because in 1.5 it tries to auth against some local server which leads to test session hanging right after passing all the tests; and there's no point in running a test like that which can _never_ be successful.

IMHO, `xfail` should be used on a test which _can_ pass if some outside conditions change, but using it here is like trying to xfail a test that tries to read `/proc/cpuinfo` on Windows - there is no point in running that test on Windows, it would _never_ be able to pass.

Reply:
makes sense